content/licensing.md

```diff
@@ -0,0 +1,18 @@
+# Licensing
+
+Licensing is about communicating what people are allowed to do with the things that you create (or what you are allowed to do with what other people create). This pertains to blogs, photographs, drawings, videos, designs, code, and research output (amongst many other things). Each piece of content has its own license. Licensing is one component to [Copyright](copyright.md).
```

Comment:
There may be a note about the difference between copyright and author's right legislations. Some things can fall under copyright in one region when it does not in another.